posix.mak

```diff
@@ -28,6 +30,12 @@
 
 QUIET:=
 
+ifeq ($(DEBUGGER),)
+DEBUGGERLOCAL=gdb
+else
+DEBUGGERLOCAL=$(DEBUGGER)
+endif
```

Comment:
Hence, `DEBUGGER=gdb` should do.
An alternative would be to allow overwriting it with environment variables as well (`DEBUGGER?=gdb`), but I think the current D Makefile style is to avoid environment variables as much as possible and thus be more explicit on the CLI.